webapp-landing/src/main/java/fi/nls/oskari/RssFeedService.java

```diff
@@ -0,0 +1,122 @@
+package fi.nls.oskari;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import fi.nls.oskari.domain.RssFeedChannel;
+import fi.nls.oskari.domain.RssFeedItem;
+import fi.nls.oskari.domain.RssFeedXmlRoot;
+import org.apache.log4j.Logger;
+import org.json.JSONObject;
+import org.json.XML;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.ClientHttpRequest;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StreamUtils;
+import java.net.URI;
+import java.nio.charset.Charset;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.List;
+import java.util.Vector;
+
+@Service
+public class RssFeedService implements Runnable {
+
+    private static final String FEED_URL_NOTIFICATIONS = "http://www.maanmittauslaitos.fi/rss/palvelutiedotteet/184";
```

Comment:
These should probably be configurable via properties. However since the landing-webapp is a separate thing I'm not sure where to put such configurations.

Reply:
Added configurable rssconfig.properties. It can be placed outside the war to {jettyhome}/resources for example.